lib/command_line_reporter.rb

```diff
@@ -118,8 +119,9 @@ def aligned(text, options = {})
              raise ArgumentError
            end
 
-    line = line.send(color) if color
-    line = line.send('bold') if bold
+    line = ANSI.public_send(color) { line } if color
```

Comment:
This code builds your case for moving the handling of colorization to a single place because it is done in a couple of places